File: Python/PRAK605-2210817220009-Azka Amalia.py
ordo=int(input())
print("Matriks A")
bil1=[];bil2=[]
for n in range(0,ordo):
    mat1=list(map(int,input().split()))
    bil1.append(mat1[0:ordo])
print("Matriks B")
for n in range(0,ordo):
    mat2=list(map(int,input().split()))
    bil2.append(mat1[0:ordo])
print("Matriks AXB")
for n in range(0,ordo):
    for m in range(0,ordo):
        hasil=0
        for o in range(0,ordo):
            hasil+=bil1[n][o]*bil2[o][m]
        print(hasil,end=' ')
    print()

File: C/PRAK605-2210817220009-Azka Amalia.c
#include<stdio.h>
int main()
{
    int mat1[20][20], mat2[20][20], hasil[20][20];char matriks;
    int a=0, n1, n2, n3, ordo;
    scanf("%d", &ordo);
    printf("Matriks A\n");
    int bil1[n1], bil2[n2];
    for(n1=0;n1<ordo;n1++){
        for(n2=0;n2<ordo;n2++){scanf("%d", &mat1[n1][n2]);}}
    printf("Matriks B\n");
    for(n1=0;n1<ordo;n1++){
        for(n2=0;n2<ordo;n2++){scanf("%d", &mat2[n1][n2]);}}
    for(n1=0;n1<ordo;n1++){
        for(n2=0;n2<ordo;n2++){
            for(n3=0;n3<ordo;n3++){
                a=a+mat1[n1][n3]*mat2[n3][n2];}
                hasil[n1][n2]=a;
                a=0;}}
                printf("Matriks AXB\n");
                for(n1=0;n1<ordo;n1++){
                    for(n2=0;n2<ordo;n2++){printf("%d ", hasil[n1][n2]);}
                        printf("\n");}
}
